assets/src/blocks/Articles/useArticlesFetch.js

```diff
@@ -59,9 +60,23 @@ export const useArticlesFetch = (attributes, postType, postId, baseUrl = null, p
 
   useEffect(() => {
     setDisplayedPosts([]);
-    loadPage(true);
+    setController(typeof AbortController === 'undefined' ? undefined : new AbortController());
```

Comment:
See [here](https://developer.mozilla.org/en-US/docs/Web/API/AbortController/abort) more information about the AbortController class